manifests/profile/postgres.pp

```diff
@@ -0,0 +1,23 @@
+class puppet_metrics_dashboard::profile::postgres (
+  $pg_password
```

Comment:
Would this be a good candidate for Sensitive? Can we enable Cert based auth, or is that too much of a reach for this initial development?